web/concrete/src/User/User.php

```diff
@@ -125,10 +127,10 @@ public function getUserInfoObject()
 
     public function __construct()
     {
-        $app = Application::getFacadeApplication();
+        $this->app = Application::getFacadeApplication();
```

Comment:
That said, if you can show with profiling that this indeed is a big improvement, it's worth having.